sound/soc/sof/ipc4-topology.c

```diff
@@ -0,0 +1,633 @@
+// SPDX-License-Identifier: (GPL-2.0-only OR BSD-3-Clause)
+//
+// This file is provided under a dual BSD/GPLv2 license.  When using or
+// redistributing this file, you may do so under either license.
+//
+// Copyright(c) 2022 Intel Corporation. All rights reserved.
+//
+//
+#include <uapi/sound/sof/tokens.h>
+#include <sound/pcm_params.h>
+#include "sof-priv.h"
+#include "sof-audio.h"
+#include "ipc4-ops.h"
+#include "ipc4-topology.h"
+#include "ops.h"
+
+/* Full volume for default values */
+#define VOL_ZERO_DB	BIT(VOLUME_FWL)
```

Comment:
VOLUME_FWL is defined at 16 but for IPC4, VOL_ZERO_DB should be 0x7fffffff right?

Reply:
> VOLUME_FWL is defined at 16 but for IPC4, VOL_ZERO_DB should be 0x7fffffff right?

Oh, yes, thanks.